apps/chat/src/types/applications.ts

```diff
@@ -134,3 +134,8 @@ export enum ApplicationType {
   CUSTOM_APP = 'custom app',
   CODE_APP = 'code app',
 }
+
+export enum Toolsets {
+  Toolset = 'toolset',
```

Comment:
it would be nice use such term `webApiToolset` everywhere instead of `toolset`
```suggestion
  WebApiToolset = 'toolset',
```